domain/model/pokemon.go

```diff
@@ -1,8 +1,7 @@
 package model
 
-type Ability struct {
```

Comment:
Document functions as others witha comment above, remember everything is being exported when Capitalized.